python_modules/dagster-graphql/dagster_graphql/schema/logs/events.py

```diff
@@ -309,14 +318,25 @@ class Meta:
     externalUrl = graphene.String()
     externalStdoutUrl = graphene.String()
     externalStderrUrl = graphene.String()
-    logManagerMetadata = graphene.String()  # TODO - replace with union?
```

Comment:
did this go out in any release? A direct type change is a breaking change that can cause cross push issues, so may need to make the new field name something different and leave this as is

Reply:
no, nothing in this PR stack is merged.